apps/jamtools/server/tracing/babel-plugin-trace-suffix.js

```diff
@@ -0,0 +1,86 @@
+// babel-plugin-auto-instrument.js
+module.exports = function ({types: t}) {
+  return {
+    visitor: {
+      Program(path, state) {
+        state.importAdded = false;
+      },
+      FunctionDeclaration(path, state) {
+        const functionName = path.node.id.name;
+        if (functionName.endsWith('_trace')) {
+          wrapFunction(path, functionName, state);
+        }
+      },
+      VariableDeclarator(path, state) {
+        if (
+          t.isIdentifier(path.node.id) &&
+          path.node.id.name.endsWith('_trace') &&
+          (t.isFunctionExpression(path.node.init) || t.isArrowFunctionExpression(path.node.init))
+        ) {
+          wrapFunction(path, path.node.id.name, state);
+        }
+      },
+      CallExpression(path, state) {
+        const callee = path.node.callee;
+        if (
+          t.isMemberExpression(callee) &&
+          (callee.object.name && (callee.object.name === 'app' || callee.object.name.toLowerCase().includes('router'))) &&
+          (callee.property.name === 'get' ||
+            callee.property.name === 'post' ||
+            callee.property.name === 'put' ||
+            callee.property.name === 'delete')
+        ) {
+          const handlerIndex = path.node.arguments.length - 1;
+          const handler = path.node.arguments[handlerIndex];
+          if (t.isFunctionExpression(handler) || t.isArrowFunctionExpression(handler)) {
+            const functionName = `Express handler - ${path.node.arguments[0].value}`;
+            const traceCall = t.callExpression(t.identifier('traceFunction'), [
+              t.stringLiteral(functionName),
+              handler,
+              t.objectExpression([
+                t.objectProperty(t.identifier('"http.method"'), t.stringLiteral(callee.property.name.toUpperCase())),
```

Comment:
Express handlers weren't being patched right by opentelemetry for some reason. This fixes that